bindings/java/src/main/com/apple/foundationdb/FDB.java

```diff
@@ -197,9 +185,25 @@ private static synchronized FDB selectAPIVersion(final int version, boolean cont
 			throw new IllegalArgumentException("API version not supported (maximum 700)");
 
 		Select_API_version(version);
-		FDB fdb = new FDB(version, controlRuntime);
+		singleton = new FDB(version);
 
-		return singleton = fdb;
+		return singleton;
+	}
+
+	/**
+	 * Disables shutdown hook that stops network thread upon process shutdown. Useful if you have your own
+	 * shutdown hook where you use FDB instance and you do not want to have race conditions
+	 * leading to accessing already closed FDB connection because all shutdown hooks are executed concurrently.
```

Comment:
A suggestion for the second sentence:

This is useful if you need to run your own shutdown hook that uses the FDB instance and you need to avoid race conditions with the default shutdown hook. It is recommend that the replacement shutdown hook stops the network thread by calling stopNetwork (note -- this should link to the function).